users/rverst/rverst.c

```diff
@@ -230,8 +230,8 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
                 register_code(KC_LGUI);
                 register_code(KC_LSFT);
                 tap_code(KC_S);
-                register_code(KC_LSFT);
-                register_code(KC_LGUI);
+                unregister_code(KC_LSFT);
+                unregister_code(KC_LGUI);
```

Comment:
And again:
```c
tap_code16(S(G(KC_S)));
```
etc.

Reply:
Oh, I must have missed that in the docs. I'll change that later, thanks.